src/schema/sources.ts

```diff
@@ -1232,8 +1232,8 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       graphorm.queryOneOrFail(ctx, info, (builder) => ({
         ...builder,
         queryBuilder: builder.queryBuilder.where(
-          `"${builder.alias}"."id" = :id`,
-          { id },
+          `(CAST("${builder.alias}"."id" AS text) = :id OR LOWER(REPLACE("${builder.alias}".title, ' ', '')) = :id)`,
```

Comment:
We need to make sure proper index is added for this

Reply:
Would we want it even if this is just the categories of 10 items? Might grow more though most likely less than a hundred.

Reply:
Yes and no, I don't know how large it will grow in the future, but either way all the casting here looks unnecessary? The `LIKE` would not work here?

Reply:
With above it feels like many queries won't really work when searching?

Reply:
Yeah, let's index it just to be sure.

> The LIKE or ILIKE would not work here?

I found that `lower` works better than ILIKE. Using `=` should be enough for us since most of our categories are single-worded. And if it is two, then the replace lower catches that.

Reply:
See my comment https://github.com/dailydotdev/apps/pull/3541#discussion_r1761686670